tests/regressions.rs

```diff
@@ -16,6 +16,7 @@ const ID1: Token = Token(1);
 const WAKE_TOKEN: Token = Token(10);
 
 #[test]
+#[cfg_attr(target_os = "hurd", ignore = "protocol not supported")]
```

Comment:
Same as above, can you expand on what protocol is not supported?

Reply:
same as above